### PR TITLE
Changed PDF file preview behavior

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Changed PDF preview behavior

## How does it look?

Screenshot 2023-12-01 at 7 35 43 p m
Screenshot 2023-12-01 at 7 35 31 p m

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/tests/permissions-7/filePreview.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      .navigateTo(`http://localhost:9090/browser`)
+      .click(bucketNameAction)
+      .click(file)
+      .click(Selector(".objectActions button").withText("Preview"))
+      .expect(Selector(".dialogContent > div > img").exists)
+      .ok();
+  })
+  .after(async (t) => {
+    //await functions.cleanUpNamedBucketAndUploads(t, bucketName);
+  });
```
Comment (cesnietor) [inline portal-ui/tests/permissions-7/filePreview.ts:55]: should we remove this?

File: portal-ui/tests/permissions-7/pdfPreview.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import * as roles from "../utils/roles";
+import { Selector } from "testcafe";
+import * as functions from "../utils/functions";
+import { namedTestBucketBrowseButtonFor } from "../utils/functions";
+
+fixture("Test PDF Display in Console").page("http://localhost:9090/");
```
Comment (cesnietor) [inline portal-ui/tests/permissions-7/pdfPreview.ts:22]: what's different between this tests and the other test file? seem familiar, if they are  not the same, can we add a bit more detail in the fixture name to quickly know the difference?

**Comment by cesnietor** [comment]
Seems that now we can't preview txt files @bexsoft. We used to can in the previous version.

**Review by cesnietor — DISMISSED** [review]

**Comment by prakashsvmx** [comment]
@bexsoft , could you fix the react warnings as reported in CI ?
may be we can exclude the file?
```
[warn] public/scripts/pdf.worker.min.js
[warn] Code style issues found in the above file. Run Prettier to fix.
```

**Review by reivaj05 — APPROVED** [review]

**Comment by bexsoft** [comment]
> @bexsoft Looks like node is installing as 16.20.2, while playwright is expecting >=18. The playwright tests did not run it seems. Did something change?

This is related to github workflow, will review in a further PR

**Review by cesnietor — APPROVED** [review]

**Pull request merged**